munit-scalacheck/shared/src/main/scala/munit/ScalaCheckSuite.scala

```diff
@@ -49,23 +49,29 @@ trait ScalaCheckSuite extends FunSuite {
 
   private def propToTry(prop: Prop, test: Test): Try[Unit] = {
     import ScalaCheckTest._
-    def seed =
+    def makeSeed() =
       scalaCheckTestParameters.initialSeed.getOrElse(
         Seed.fromBase64(scalaCheckInitialSeed).get
       )
+    val initialSeed = makeSeed()
```

Comment:
This should be the actual "initial seed", one that's used to generate seeds for all test runs (default 100) of the same property.